src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const dispatch = useDispatch();
+
+  const resetCurrentTodo = () => {
+    dispatch(setCurrentTodo(null));
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(todo.userId)
+      .then(userByTodo => setUser(userByTodo))
+      .finally(() => setIsLoading(false));
+  }, []);
```

Comment:
React Hook useEffect has a missing dependency: 'todo.userId'. Either include it or remove the dependency array